src/main/java/bio/terra/service/job/JobService.java

```diff
@@ -102,6 +106,8 @@ public JobService(
    * encapsulates all Stairway interaction.
    */
   public void initialize() {
+    SentryUtils.initializeSentry(
```

Comment:
Should we only initialize if Sentry is configured?  E.g. if it's null I think this cause problems maybe?

Reply:
Sentry has logic to handle the case that the DSN string is empty and it only initializes when the dsn is a defined string. I've set the default in application properties to be an empty string. This seems to be the suggested workflow and what I've seen across other repos. I'm inclined to let Sentry handle this logic. 

However, it does not handle the case that SENTRY_DSN is completely undefined/null.  If I'm defining SENTRY_DSN in application.properties, is it possible for it to ever be undefined?

Reply:
I've added some null handling for the dsn!